presto-spi/src/main/java/io/prestosql/spi/security/Identity.java

```diff
@@ -29,6 +33,7 @@
     private final Optional<Principal> principal;
     private final Map<String, SelectedRole> roles;
     private final Map<String, String> extraCredentials;
+    private final Set<String> groups;
```

Comment:
Let's make the order in this class and `ConnectorIdentity` be consistent. Move this to be after `user` everywhere: fields, constructor, assignment, getter, toString, builder.

The full order is user, groups, principal, roles, extraCredentials.